join/config.toml

```diff
@@ -396,28 +396,27 @@ namespace = "tendermint"
 [axelar_bridge_btc]
 
 # Address of the bitcoin RPC server
-rpc_addr    = "<your-bitcoin-rpc-and-port>"
+# chain maintainers must set their own rpc endpoint
+rpc_addr=""
 
 # Timeout period in which the bitcoin node is expected to start (and provide the auth cookie)
 timeout_startup = "60s"
 
 # Timout period after startup in which the bitcoin node is expected to start servicing RPC calls
 timeout_rpc = "100s"
 
-# For testing, the connection to the btc bridge can be ignored.
-# In that case, all functionality requiring access to Bitcoin data is mocked.
-start-with-bridge = true
-
 ##### EVM bridges options #####
 [[axelar_bridge_evm]]
 
 # Chain name
 name = "Ethereum"
-# Address of the ethereum RPC proxy
+
+# Address of the ethereum RPC server
+# chain maintainers must set their own rpc endpoint
 rpc_addr    = "<your-ethereum-rpc>"
```

Comment:
Why still `<your-ethereum-rpc>` and not the empty string?  Won't this will continue to cause confusion?

Reply:
yeah, maybe we should have removed this as well. But for EVM at least we still have the `start-with-bridge` flag, which is currently set to false. So this shouldn't be a problem.

Reply:
also, there is another thing to consider: bitcoin is currently inactive, so we don't even want validators to set up rpcs. This way we are encouraging them to set them up explicitly for ethereum. In any case, we might need to update this config file for more EVM chains, so we can get back to this later.